tests/base/parser/bad/gold/type_t-map-cid-lparen-map-cid-cid-type.scilla.gold

```diff
@@ -1,11 +1,9 @@
 {
   "errors": [
     {
-      "error_message":
-        "This is an invalid map type, the map value type is likely incorrect. The map type must be a storable type, i.e., a base type, a valid ADT, or a map type.\n",
+      "error_message": "Invalid type or pattern. If this is a type, then this is likely due to an illegal map value type. If this is a pattern-match, then it is likely due to a misplaced or missing double arrow ('=>').\n",
```

Comment:
This is the error message from which I think we can now remove the "If this is a pattern-match.." sentence, at which point it is basically equivalent with the previous error message.

Reply:
I think we should leave it as it is a different pattern that can be used in different contexts that is causing the error. The remaining part of the error message might be helpful for some of the other cases.